yarl/__init__.py

```diff
@@ -446,7 +447,10 @@ def host(self):
             # fe80::2%Проверка
             # presence of '%' sign means only IPv6 address, so idna is useless.
             return raw
+        return self._get_host_decoded(raw)
 
+    @lru_cache()
```

Comment:
The default size for `@lru_cache()` is 128: https://docs.python.org/3/library/functools.html#functools.lru_cache
`re` module has 512 default for caching.

I have no strong opinion for `yarl` cache size but would like to discuss it at least.

Reply:
We can use 512 as `re` module. The perfect size is application dependant, we could try to make this configurable by doing an cache decorator and getting the cache size by a variable configuration.

What do you think?

Reply:
Let's not overcomplicate from the start: use 512 non-configurable constant.
We can add customization later on people's demand.